content/en/blog/2021/direct-upgrade/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Announcing Support for 1.8 to 1.10 Direct Upgrades"
+description: "Moving Towards a Smoother Upgrade Process."
+publishdate: 2021-05-24
+attribution: "Mitch Connors (Google), Sam Naser (Google)"
+keywords: [upgrade,Istio,revision]
+---
+
+As Service Mesh technology moves from cutting edge to stable infrastructure many users have expressed an interest in upgrading their service mesh less frequently, as qualifying a new minor release can take a lot of time.  This can be especially difficult for users who don’t keep up with new releases, as Istio has not supported upgrades across multiple minor versions.  To upgrade from `1.6.x` to `1.8.x`, users first had to upgrade to `1.7.x`.
+
+With the release of Istio 1.10, we are announcing (Alpha|Experimental) level support for upgrading directly from Istio `1.8.x` to `1.10.x`, without upgrading to `1.9.x`.  We hope this will reduce the operational burden of running Istio, in keeping with our 2021 theme of improving Day 2 User Experience.
+
+## Upgrade From 1.8 to 1.10
+
+For direct upgrades we recommend using the canary upgrade method so that control plane functionality can be verified before cutting workloads over to the new version. We’ll also be using revision tags in this guide, an improvement to canary upgrades that was introduced in 1.10 to allow users to label their namespaces once with stable identifiers.
```

Comment:
I don't think they label once with stable indentifiers. I think they do label once, but it might be stable or pre prod, or some other label.

Reply:
I guess it depends what stable means here? They're stable in the sense that once labeled the labels don't have to be changed.

Reply:
Not sure if we have links from blogs to other blogs, but could adda link in that sentence to https://istio.io/latest/blog/2021/revision-tags/.